rust/platform/triple.bzl

```diff
@@ -117,7 +117,7 @@ def get_host_triple(repository_ctx, abi = None):
     # Detect the host's cpu architecture
 
     supported_architectures = {
-        "linux": ["aarch64", "x86_64", "s390x"],
+        "linux": ["aarch64", "x86_64", "s390x", "powerpc64le"],
```

Comment:
Is the host platform ever gonna be powerpc? I think the only changes you need are what was done in `rust/platform/triple_mappings.bzl`.

Reply:
If I understand correctly, the host platform is the platform the bazel binary is executed on. When attempting to utilize rules_rust and executing bazel on a ppc64le host, I did receive the failure message from `_validate_cpu_architecture`, causing the build to fail. Marking it as supported here seems necessary to avoid failure in that case.